qiskit/transpiler/target.py

```diff
@@ -969,6 +970,18 @@ def __setstate__(self, state: tuple):
         self._instruction_schedule_map = state["instruction_schedule_map"]
         super().__setstate__(state["base"])
 
+    def convert_to_dt(self, duration: float) -> int:
+        """Convert a given duration in seconds to units of dt
+
+        Args:
+            duration: The duration in seconds, such as in an :class:`.InstructionProperties`
+                field for an instruction in the target.
+
+        Returns
+            duration: The duration in units of dt
+        """
+        return duration_in_dt(duration, self.dt)
+
```

Comment:
This would need a feature release note.  Perhaps it ought to be `seconds_to_dt` so it's obvious at the call site what unit it expects too, rather than being documentation only.

Reply:
I thought in this context of `Target` all being in terms of seconds it was implicit. But I can change the name if you think it's more clear.

Reply:
Renamed in: https://github.com/Qiskit/qiskit/pull/13506/commits/78a7ee49b8b67e0c7a4776380fc4ace65e1ea419